src/fides/api/schemas/privacy_center_config.py

```diff
@@ -15,7 +15,7 @@ class IdentityInputs(FidesSchema):
 
 class CustomPrivacyRequestField(FidesSchema):
     label: str
-    required: Optional[bool] = False
+    required: Optional[bool] = True
```

Comment:
Updating this to default `required` to `True` if not provided to match the current Privacy Center behavior.